bin/req_packages_anovos_py_3_8_x.sh

```diff
@@ -1,19 +1,4 @@
-sudo pip3 install --upgrade cython
```

Comment:
@VishnuGowthem Do you happen to know why there is a script for Python 3.8 when we use 3.7 for Spark 2.4.x and 3.9 for Spark 3.x during testing?